test/integration/targets/expect/tasks/main.yml

```diff
@@ -241,3 +241,106 @@
     that:
     - "non_zero_result.failed"
     - "non_zero_result.msg == 'non-zero return code'"
+
+- name: run tests in venv
+  vars:
```

Comment:
Not necessary, but if you want to get fancy, you could actually add `module_defaults` to this block to avoid passing `virtualenv` to all the `pip` module invocations...